dags/eoy_create_tables.py

```diff
@@ -136,18 +128,6 @@ def congestion_create_table(dt):
                                     dag = dag,
                                     op_args = ['{{ ds }}'])
 
-wys_create_table = PythonOperator(task_id='wys_create_table',
-                                    python_callable = create_wys_raw_data_table,
-                                    dag = dag,
-                                    op_kwargs = {'pg_hook': wys_bot,
-                                                 'dt': '{{ ds }}'}
-                                    )
-wys_replace_trigger = PythonOperator(task_id='wys_replace_trigger',
-                                    python_callable = replace_wys_raw_data_trigger,
-                                    dag = dag,
-                                    op_kwargs = {'pg_hook': wys_bot,
-                                                 'dt': '{{ ds }}'})
-
```

Comment:
This [line](https://github.com/CityofToronto/bdit_data-sources/blob/bdb853f4fb869c15e795aa9e906cfba46bb1d6ef/dags/eoy_create_tables.py#L146) needs to be deleted as well. Also check if any of these deleted variables is used anywhere in the file.